src/App.js

```diff
@@ -100,32 +100,27 @@ import ElevatteLandingPage from "demos/ElevatteLandingPage";
 // import PrivacyPolicyPage from "pages/PrivacyPolicy.js";
 
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import ReactGA4 from 'react-ga4';
 import Hotjar from '@hotjar/browser';
+import { sendPageView } from "helpers/GoogleAnalytics";
 
 const siteId = 5033364;
 const hotjarVersion = 6;
 Hotjar.init(siteId, hotjarVersion);
 
-const GoogleAnalyticsTrack = () => {
-  ReactGA4.initialize('G-LCLW13KEKW');
-  ReactGA4.send({ hitType: "pageview", page: window.location.pathname + window.location.search, title: document.title });
-}
-
 export default function App() {
   // If you want to disable the animation just use the disabled `prop` like below on your page's component
   // return <AnimationRevealPage disabled>xxxxxxxxxx</AnimationRevealPage>;
-
+ 
   return (
     <>
       {/* to fix */}
-      {window.location.hostname === 'elevatte.me' && <GoogleAnalyticsTrack />}
+      {window.location.hostname === 'elevatte.me' && sendPageView()}
       <GlobalStyles />
       <Router>
         <Routes>
           {/* <Route path="/components/:type/:subtype/:name" element={<ComponentRenderer />} /> */}
           {/* <Route path="/components/:type/:name" element={<ComponentRenderer />} /> */}
-          <Route path="/" element={<ElevatteLandingPage />} />
+          <Route path="/" element={< ElevatteLandingPage />} />
```

Comment:
Não tem necessidade desse espaço

```suggestion
          <Route path="/" element={<ElevatteLandingPage />} />
```